airflow/serialization/serialized_objects.py

```diff
@@ -651,7 +651,7 @@ class SerializedDAG(DAG, BaseSerialization):
     not pickle-able. SerializedDAG works for all DAGs.
     """
 
-    _decorated_fields = {'schedule_interval', 'default_args', '_access_control'}
+    _decorated_fields = {'schedule_interval', 'default_args', 'params', '_access_control'}
```

Comment:
Can you add a test in https://github.com/apache/airflow/blob/79c1a0e3a248a379a5d5338fe41675104a241595/tests/serialization/test_dag_serialization.py#L556-L601 please